ObjectiveCloudant/Operations/CDTQueryViewOperation.m

```diff
@@ -0,0 +1,291 @@
+//
+//  CDTGetViewOperation.m
+//  ObjectiveCloudant
+//
+//  Created by Rhys Short on 15/02/2016.
+//  Copyright (c) 2016 IBM Corp.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file
+//  except in compliance with the License. You may obtain a copy of the License at
+//    http://www.apache.org/licenses/LICENSE-2.0
+//  Unless required by applicable law or agreed to in writing, software distributed under the
+//  License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+//  either express or implied. See the License for the specific language governing permissions
+//  and limitations under the License.
+//
+#import "CDTQueryViewOperation.h"
+
+static const int kCDTDefaultOperationIntegerValue = -1;
+
+@implementation CDTQueryViewOperation
+
+- (instancetype)init
+{
+    self = [super init];
+    if (self) {
+        _descending = NO;
```

Comment:
For consistency, we should probably either explicitly set `_reduce` to `NO` in this `if` statement or remove the other `BOOL`s that are being explicitly set to `NO` as `NO` is the default for a `BOOL` instance variable.